src/ejabberd_sm.erl

```diff
@@ -484,14 +487,19 @@ init([]) ->
                   end,
                   ?ALL_HOST_TYPES),
     %% Create metrics after backend has started, otherwise probe could have null value
-    create_metrics(),
+    start_probes(),
```

Comment:
I think such names are short-lived, because you are assuming that all global metrics are probes, and an addition of a new metric would change that. I'd rather change line 488 to have `[global | ?ALL_HOST_TYPES]`.